### ISSUE TITLE
RNG help, anyone?

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/DHClass-Hub#420]
@Pomilui I noticed you'd pushed some files to Troubleshooting here on DHClass-Hub earlier today...do you want some help with debugging them or was that just a trial push? 

When you want help with debugging, it helps to open an issue here, and then patch in a link to the files in Troubleshooting, and tell us what you're having problems with...Sometimes the act of explaining the problem helps you see how to solve it, and sometimes you just need some others' eyes to help you spot the problem!

@ebeshero @jonhoranic @alexfell06 @MarkTheShark01 @dorothealint @Pomilui @garrettjoiner @quantum-satire

**Comment by Pomilui** [comment]
@ebeshero Yep.  Created the files directly on the troubleshooting page from GitHub, so it wasn't a test push.  I'll be sure to open an issue and put a link in the post next time.  I'll have to go over that assignment again achieve maximum validity.

**Comment by Pomilui** [comment]
Hey, all.  I'm getting a syntax error on line 11 of this file: https://github.com/ebeshero/DHClass-Hub/blob/master/TroubleShooting/pcsolar-Relax_x2-debug.   It reads "The document is not well formed".  Is there something I forgot to define above, or is there something that I need to add?  Any help would be appreciated.  Thanks.  - Nick  @ebeshero @jonhoranic @alexfell06 @MarkTheShark01 @dorothealint @Pomilui @garrettjoiner @quantum-satire

**Comment by ebeshero** [comment]
Hi Nick--Great--I think @quantum-satire  worked on debugging the relax NG files you submitted yesterday, so those will come back pretty soon in the homework returns--stay tuned! I'm taking a look at your file now...

**Comment by ebeshero** [comment]
@Pomilui Ahh! That was hard to spot at first, but I saw it better when I opened it in oXygen. The error is on this line:
```
topic = attribute topic ("agree" | "conditions" | "meet"}
```
You've started defining the content of the attribute topic with a left parenthesis `(` instead of a left curly brace `{`

Note: Make sure you save this with a file extension, and for this you want `.rnc` (Otherwise it's a bit hard to get this open with the right file type, and might cause trouble with associating the schema, etc.)

**Comment by Pomilui** [comment]
Progress has been made, but I've been fiddling with the set of validity errors from this rnc file:  https://github.com/ebeshero/DHClass-Hub/blob/master/TroubleShooting/pcsolar-Relax_x2-debug
I think it stems from line 9 or lines 12-15:   **ship, carrier, and receiver** are "not allowed here; expect element end tag or text".  I tried putting having mixed syntax grouping for condition with no luck.  Thanks for the help so far.  I don't know if I should still be pinging everybody, but oh well.
 - Nick
@ebeshero @jonhoranic @alexfell06 @MarkTheShark01 @dorothealint @Pomilui @garrettjoiner @quantum-satire

**Comment by ebeshero** [comment]
@Pomilui If you ping once in the Issue thread it'll reach all of us anyway--no worries either way, but you don't have to. I typically use the `@` to ping the person I'm talking to by default. Tomorrow in class we can test if you're all notified by e-mail whether we use the class "ping" group lines or not. I think I tend to set those up for assignments to direct everyone's attention, but they may not be necessary...

**Comment by Pomilui** [comment]
Hey, again, all.
   I worked out the mixed syntax grouping for the three elements within the condition element.  All good now.  See you tomorrow.

**Comment by ebeshero** [comment]
@Pomilui Glad you worked this out...Any lingering issues? I'll go post the upload link!

**Comment by ebeshero** [comment]
By the way, @jonhoranic and I were in class this afternoon and this evening, which meant neither of us were able to help debug for a few hours...but others should chime in to help!

**Comment by ebeshero** [comment]
There! The homework submission link for tomorrow is up!
